librrd.c

```diff
@@ -134,16 +134,17 @@ json_for_source(RRD_SOURCE * source)
     json_object_set_string(src, "default",
                            source->rrd_default ? "true" : "false");
 
-    char           *owner = NULL;
+    char           owner[128];
+
     switch (source->owner) {
     case RRD_HOST:
-        owner = "host";
+        snprintf(owner, sizeof owner, "host");
         break;
     case RRD_VM:
-        owner = "vm";
+        snprintf(owner, sizeof owner, "vm %s", source->owner_uuid);
```

Comment:
I think to prevent buffer overruns and make coverity happy, we should consider the following:
```
owner[128] = {0};
...
sprintf(owner, sizeof(owner)-1, "vm %s", source->owner_uuid))
```

However, we'd only ever run into this, if the library user provides a too long uuid.